cmd/openshift-install/create.go

```diff
@@ -353,6 +353,10 @@ func waitForInitializedCluster(ctx context.Context, config *rest.Config) error {
 	}
```

Comment:
wdyt:
```
if err != nil {
    if err == wait.ErrWaitTimeout && lastErr != "" {
        err = errors.New(lastErr)
    }
   return errors.Wrap(err, "failed to initialize the cluster")
}

return nil
```

Reply:
> ```go
> if err != nil {
>     if err == wait.ErrWaitTimeout && lastErr != "" {
>         err = errors.New(lastErr)
>     }
>    return errors.Wrap(err, "failed to initialize the cluster")

This leaves `lastErr` out of the message if we die for a reason besides a timeout (e.g. if the caller decides to cancel the `Context`, or `UntilWithSync` freaks out for whatever reason).  I'd rather include `lastErr` whenever it was non-empty.

Reply:
in that case do we care about the err from `UntilWithSync` at all. always `lastErr` it it is not empty..?